README.md

```diff
@@ -66,6 +66,38 @@ module.exports = {
 }
 ```
 
+### Minimizing For Production
+
+While webpack 5 is likely to come with a CSS minimizer built-in, with webpack 4 you need to bring your own. To minify your the output, use a plugin like [optimize-css-assets-webpack-plugin](https://github.com/NMFR/optimize-css-assets-webpack-plugin):
```

Comment:
@mike-marcacci Hey, it's me again. Sorry to be picky, but I think you still have a typo:
"To minify ~your~ the output"
I think you want either "your" or "the", but not both.